dashboard/app/models/levels/dsl_defined.rb

```diff
@@ -149,7 +149,7 @@ def filename
 
   def existing_filename
     # Find a file in config/scripts/**/*.[class]* containing the string "name '[name]'"
-    grep_string = "grep -lir \"name '#{name}'\" --include=*.#{self.class.to_s.underscore}* config/scripts --color=never"
+    grep_string = "grep -lir \"name '#{name}'\" --include=*.#{self.class.to_s.underscore}* #{Rails.root}/config/scripts --color=never"
```

Comment:
This was needed because the script is being run from the root of the repo, not the dashboard directory. I believe this is a safe change.